microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/InterceptorSimplyTimed.java

```diff
@@ -43,4 +56,24 @@ void postComplete(SimpleTimer metric) {
         metric.update(duration());
     }
 
+    @Inherited
+    @InterceptorBinding
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @Metered
+    @interface Binding {
+        class Literal extends AnnotationLiteral<Binding> implements Binding {
+
+            private static final long serialVersionUID = 1L;
+
+            private static final Literal INSTANCE = new Literal();
+
+            static Literal getInstance() {
```

Comment:
please rename to `instance` (in all interceptors)

Reply:
Changed